tests/pkg_tweetnacl/tests/01-run.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    board = os.environ['BOARD']
+    # Increase timeout on "real" hardware
+    timeout = 120 if board is not 'native' else -1
+    child.expect('OK \(\d+ tests\)', timeout=timeout)
```

Comment:
`flake8` now complains about this one. It should be a regular expression string

```suggestion
    child.expect('OK \(\d+ tests\)', timeout=timeout)
    child.expect(r'OK \(\d+ tests\)', timeout=timeout)
```

Reply:
Why is it suddenly complaining here, but not with the other PRs similar to this one?

Reply:
It is a really old issue they finally managed to detect, I see they released a new version 10hours ago and `travis` is installing the pip version.

https://gitlab.com/pycqa/flake8/issues/465

Reply:
Ah, that explains :)